apps/admin-gui/src/app/shared/components/user-detail-page/user-groups/user-groups.component.html

```diff
@@ -1 +1,25 @@
-<p>user-groups works!</p>
+<app-refresh-button (refresh)="refreshTable()"></app-refresh-button>
+
+<h1 class="page-subtitle pt-2">{{'SHARED.COMPONENTS.USER_DETAIL.GROUPS.MEMBER_GROUPS' | translate}}</h1>
+
+<mat-spinner class="ml-auto mr-auto" *ngIf="loading"></mat-spinner>
+<div *ngIf="!loading">
+  <app-groups-list
```

Comment:
Could you please add an optional input parameter for the group list representing the default page size and set it here to 5? It would look better with the two tables.